### DISCUSSION TITLE
동아리 프로젝트 네이밍 규칙

### DISCUSSION THREAD

**Discussion by Cotidie** [ApptiveDev/.github#6] [Ideas]
## 개요
현재 `apptive-<기수>-<프로젝트명>-<프론트/백엔드>`가 규칙인데
apptive와 기수는 태그(토픽)으로 대체하고자 합니다.

## 개선
- 네이밍: `프로젝트명-프론트/백엔드` 및 apptive, 기수 태그
  - 영문은 파스칼 케이스 또는 모두 소문자 중 선택
  - 띄어쓰기는 하이픈(-)으로 대체
- 예시:  `EasyWind-Backend`, `EasyWind-Frontend`

## 소문자 이유?
주된 이유는 대소문자 구분에서 자유롭기 위함입니다. 대문자가 섞일 경우 경우의 수가 많기 때문에 검색에 불편할 수 있습니다.
- url 접근이 용이함
   - ex) `github.com/easywine-frontend`, `github.com/Easywind-Frontend`, `github.com/EasyWine-Frontend` 등..
- 파일 시스템 등에서 찾기 용이함
   - ex) easywine, Easywine, EasyWine 등 대소문자 구분에 따라 검색되지 않는 경우가 있음.

그러나 보통 대소문자 구분을 하지 않도록 설정할 수 있기 때문에 큰 문제는 아닙니다.

**Comment by Kick-snare** [comment]
동의합니다. 기존 방식의 네이밍은 가독성도 떨어지고 간결하지 못하네요
